app/views/reservations/_reservations_list.html.erb

```diff
@@ -13,18 +13,18 @@
   </thead>
   <tbody>
 
-  <% reservations_set.includes(:equipment_model, :equipment_object).each do |reservation| %>
+  <% reservations_set.includes(:equipment_model, :equipment_object).find_each do |reservation| %>
     <tr>
       <td><%= link_to reservation.id, reservation %></td>
       <% if can? :manage, Reservation %>
         <td>
         <div class="dropdown">
             <%= link_to reservation.reserver.name, '#', :"data-toggle" => 'dropdown' %>
-            <% if (can? :manage, Reservation) && reservation.reserver.id %>
+            <% if reservation.reserver_id %>
```

Comment:
The whole block can only ever get accessed if you already are a checkout person, so I got rid of the redundant check